src/video.tsx

```diff
@@ -238,8 +248,47 @@ const Video = React.forwardRef<HTMLVideoElement, VideoProps>(
                 width={props.width}
                 height={props.height}
             >
+                <div
+                    style={{
+                        position: 'absolute',
+                        width: '100%',
+                        height: '100%',
+                    }}
+                    ref={(element) => {
+                        const containerElement = element ? (element as VideoElement) : null;
+
+                        if (containerElement) {
+                            containerElement.play = () => videoRef.current?.play() || new Promise(() => {});
```

Comment:
why do you return a never resolving promise here?

Reply:
fixed